assets/_scss/elements/_typography.scss

```diff
@@ -116,7 +116,7 @@ dfn {
 }
 
 .usa-sans,
-.usa-sans p {
+.usa-sans * {
```

Comment:
What about lists and spans? Should we add them too? I didn't catch them so added the wildcard.